prisma/schema.prisma

```diff
@@ -0,0 +1,33 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
+// Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider  = "postgresql"
+  url       = env("DATABASE_URL")
+  directUrl = env("DIRECT_URL")
+}
+
+model User {
+  id                  String                  @id @default(uuid())
+  name                String
+  email               String                  @unique
+  password            String
+  created_at          DateTime
+  updated_at          DateTime
+}
+
+model UserAuthProvider {
+  id            String   @id @default(uuid())
+  user        User @relation(fields: [user_id], references: [id])
```

Comment:
@IrinaStoetski have you tested these changes? 
My suggestion was just from top of my mind, not tested be me.